### DISCUSSION TITLE
How to autoconnect bluetooth-devices on startup?

### DISCUSSION THREAD

**Discussion by PeterKriegtNenWolf** [Vanilla-OS/live-iso#377] [Q&A]
Hello, 
I would like  a bluetooth-device to connect automatically on startup of the system. Unfortunately, I was not able to find a way to achieve this. I tried following this guide (https://wiki.debian.org/BluetoothUser) and similar approaches from here (https://askubuntu.com/questions/17504/how-can-i-have-a-bluetooth-keyboard-auto-connect-at-startup) and running commands like "hcitool" but none of the commands were found by the shell and i struggled installing them using "apt-get install", as they were also not able to be located. 
Any idea, how i can interact with the bluetooth-packages that vanilla is using to achieve this?

**Comment by CarlAyoub** [comment]
I am having the same kind of trouble. Windows autoconnects as soon as I click on the keyboard but VanillaOS sometimes does not want to at all. The only way to connect would be to force it to by clicking connect in the bluetooth menu. 

I heard the way to do it is to "trust" the devices. However the gui in settings it too basic and I tried everything but hci does not wanna work, and nothing else wants to. I tried installing Blueman but it does not seem to have access to the devices at all. I tried installing Bluez but it also didn't work.

Any help is greatly appreciated.
